test/cpp/binding.gyp

```diff
@@ -3,6 +3,7 @@
     {
         "cflags" : ["-Wall", "-Wextra", "-Wno-unused-parameter"],
         "defines": [ "V8_DEPRECATION_WARNINGS=1" ],
+        "libraries": [ "../../../build/Release/node_sqlite3.node" ],
```

Comment:
I tested it but it doesn't seem to work on Mac. Could you try building with this instead:
```
     "libraries": [ "-L../../../build/<(PRODUCT_DIR)" ],
```
(I.e. only set the path to look for the library in the right place?)